### ISSUE TITLE
Post button doesn't have an icon

### ISSUE THREAD

**Issue by Fxsch** [thunder-app/thunder#1537]
### Bug Description

Just updated to 0.5.0 and when I created a new post, I noticed that the post button (I like the new position of it btw) was just empty without an icon. The icon shows when the post is empty but as soon as you type and the button changes color, the icon is gone.

### Expected Behaviour

Icon always visible

### Steps to Reproduce

1. Click on the **+** to create a new post
2. Type something
3. Icon is invisible

### Additional Context

Here's a quick recording of how that looks:

https://github.com/user-attachments/assets/bb98f150-57d4-4efd-995f-b40172bffc24

### App Version

0.5.0

### Device

Nothing Phone 1

### OS

Android 13

**Comment by Asudox** [comment]
The icon is definitely not gone. I am not sure how you can't see it but because the background color is so bright, the icon appears to be "invisible" when in reality it just blends in with the bright background color you have.

**Comment by Fxsch** [comment]
You're right. I tested it with different wallpapers but I noticed it was always just barely visible. When I turned off dark mode, it was easily visible because then the button color is a lot darker. It makes sense to have a dark button color with a bright background and a bright button color with a dark background. I think the icon should just be made black when using dark mode, that would solve the issue.
